### DISCUSSION TITLE
is it possible to continue with the test rest of the test if a step fails for Cypress ?

### DISCUSSION THREAD

**Discussion by mkhizar53** [cypress-io/cypress#18084] [Questions and Help]
is it possible to continue with the rest of the test if a step fails in Cypress - I'm using cucumber and cypress together. I want to delete multiple elements within my dom and after it deletes all the items, it still looks for the element and outputs a failure. I would like to ignore the failure and move on with the rest of the scenario. How can i do that?

**Comment by Adolfo-Macias** [comment]
Hi @mkhizar53 according to the issue https://github.com/cypress-io/cypress/discussions/9449, its not possible to continue.
